sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -4,12 +4,6 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
 - Allow HTTP connections
 
 ## 12.23.0-beta.1 (2023-11-01)
```

Comment:
This version change log might should move below the one release on 11/9.
If so, the change log "Allow HTTP connections" looks not valid.